iam-login-service/src/main/webapp/resources/iam/apps/dashboard-app/components/user/password/user.password.component.js

```diff
@@ -50,7 +50,7 @@
         actionButtonText: 'Send password reset e-mail',
         headerText: 'Send password reset e-mail',
         bodyText:
-            `Are you sure you want to send the password reset e-mail to ${self.user.name.formatted}?`
+            `Are you sure you want to send the password reset e-mail to ${self.user.name.formatted} at ${self.userCtrl.user.emails[0].value}?`
```

Comment:
I think also `${self.user.name.formatted}` should be `${self.userCtrl.user.name.formatted}` to display the updated name correctly (e.g. if you change name or surname and then send an email to reset the password without refreshing the page)